project.clj

```diff
@@ -38,10 +95,17 @@
                       :dependencies [[org.clojure/clojure "1.11.0-master-SNAPSHOT"]
                                      [org.clojure/clojure "1.11.0-master-SNAPSHOT" :classifier "sources"]]}
 
-             :test {:resource-paths ["test-resources"]
+             :test {:dependencies [[org.clojure/java.classpath "1.0.0"]]
+                    :resource-paths ["test-resources"]
                     ;; Initialize the cache verbosely, as usual, so that possible issues can be more easily diagnosed:
                     :jvm-opts ["-Dorchard.initialize-cache.silent=false"]}
 
+             :no-dynapath {:jvm-opts ["-Dorchard.fetch-java-sources-via-dynapath=false"]
+                           :resource-paths [~(unzipped-jdk-source)]
+                           :plugins ~(if jdk8?
+                                       '[[lein-jdk-tools "0.1.1"]]
```

Comment:
Ref: https://github.com/pallet/lein-jdk-tools

Turned out to be necessary.

Reply:
Why so?

Reply:
> Why so?

It compensates, when needed, for the loss of this functionality: https://github.com/clojure-emacs/orchard/blob/f763175a4e8f7b8d7e6df5be1472b3fd9c6ec757/src/orchard/java.clj#L73-L81

Reply:
Got it.